static/src/javascripts/projects/commercial/modules/dfp/create-slot.ts

```diff
@@ -152,6 +152,8 @@ const adSlotConfigs: AdSlotConfigs = {
 	},
 };
 
+export const AD_SLOT_ID_PREFIX = 'dfp-ad--';
```

Comment:
We have `adSlotSelector` in `dfp-env`:

https://github.com/guardian/frontend/blob/0093a5e35b658a3dba9fdb34ccb4a0bf310f7a64/static/src/javascripts/projects/commercial/modules/dfp/dfp-env.ts#L28

As you need to distinguish inline, I'm only really pointing this out as perhaps a more central place for the selector const to live rather than `create-slot`?

Also maybe it's better to use classes like `ad-slot` and `ad-slot--inline` rather than an id? The queries could then simplify to something like:

```js
document.querySelectorAll('.ad-slot')
document.querySelectorAll('.ad-slot--inline')
```

We'd need to be sure however that those classes are present for all ad types.

Reply:
I like the idea of using `.ad-slot` rather than the begging of the elements' `id`s. 
It looks like each page type gives its inline ad slots different classes so we'd have to do something like 
```
document.querySelectorAll('.ad-slot--inline,.ad-slot--liveblog-inline,.ad-slot--container-inline,.ad-slot--gallery-inline'); // etc.
```

Reply:
Oh thats annoying. Maybe we could ensure `ad-slot-inline` is on all inline ads?

Could we do that by adding a default class to:

https://github.com/guardian/frontend/blob/0fc0f0220d407f7326fa213a8676c56d04c0bcf5/static/src/javascripts/projects/commercial/modules/dfp/create-slot.ts#L296

Then as the default case would be handled by the above, we can remove from:

https://github.com/guardian/frontend/blob/0fc0f0220d407f7326fa213a8676c56d04c0bcf5/static/src/javascripts/projects/commercial/modules/article-body-adverts.ts#L133

Non-blocking though! It could be a follow up ticket.

Reply:
Adding `ad-slot-inline` seems like a good idea but it might be more involved than just a change to `create-slot`. It looks like some inline slots like for galleries are [inserted server-side](https://github.com/guardian/frontend/blob/b7bf23e267c2599c186d300ef40cf239688ca326/applications/app/views/fragments/gallerySlot.scala.html#L17) so don't go through `createAdSlot` and some ads that are created with `createAdSlot` like mobile sticky aren't inline.
I vote we follow up with another ticket.

Reply:
Ah great find, yes it feels like a bigger and separate piece of work.

Maybe 'inline' is not quite correct and we want to distinguish between 'fixed' and 'dynamic' or SSR and CSR. Anyway a question for another ticket.